core/mo-services-impl/ccsds-com-impl/src/main/java/esa/mo/com/impl/archive/db/PreparedStatements.java

```diff
@@ -17,7 +17,7 @@ public PreparedStatements(Connection serverConnection) {
     private static String SELECT_COM_OBJECTS = "SELECT objectTypeId, domainId, objId, "
```

Comment:
If the statement cannot be statically resolved, it is not a prepared statement anymore